src/taipy/config/_json_serializer.py

```diff
@@ -22,7 +25,9 @@ class _JsonSerializer(_BaseSerializer):
     @classmethod
     def _write(cls, configuration: _Config, filename: str):
         with open(filename, "w") as fd:
-            json.dump(cls._str(configuration), fd, ensure_ascii=False, indent=0, check_circular=False)
+            json.dump(
+                cls._str(configuration), fd, ensure_ascii=False, indent=0, check_circular=False, cls=_CustomEncoder
```

Comment:
Since `cls._str(configuration)` already convert `datetime.datetime` and `datetime.timedelta` to string, do we need the encoder here?